app/Models/Extensions/ThrowsConsistentExceptions.php

```diff
@@ -138,8 +140,9 @@ public function toJson($options = 0): string
 			// that this method does so.
 			// Hence, we call `json_encode` _without_ specifying
 			// `JSON_THROW_ON_ERROR` and then mimic that behaviour.
+			// TODO: VERIFY THIS
 			$json = json_encode($this->jsonSerialize(), $options);
-			if (json_last_error()) {
+			if ((bool) json_last_error()) {
```

Comment:
```suggestion
			if (json_last_error() !== JSON_ERROR_NONE) {
```

Just my attempt to avoid boolean casts.